source/Lib/CommonLib/InterPrediction.cpp

```diff
@@ -2093,8 +2097,9 @@ void InterPrediction::xFillIBCBuffer(CodingUnit &cu)
 void InterPrediction::xIntraBlockCopy( PredictionUnit &pu, PelUnitBuf &predBuf, const ComponentID compID )
 {
   const unsigned int lcuWidth = pu.slice->getSPS()->getMaxCUWidth();
-  int shiftSample = ::getComponentScaleX( compID, pu.chromaFormat );
-  const int ctuSizeLog2 = getLog2(lcuWidth) - shiftSample;
+  const int shiftSampleHor = ::getComponentScaleX( compID, pu.chromaFormat );
+  const int shiftSampleVer = ::getComponentScaleY( compID, pu.chromaFormat );
+  const int ctuSizeLog2 = getLog2(lcuWidth) - shiftSampleVer;
```

Comment:
Same as above, could this be ctuSizeVerLog2?

Reply:
Thank your comments. I have been modified and rebase my patch.